enterprise/models.py

```diff
@@ -94,21 +109,33 @@ class EnterpriseCustomerUser(TimeStampedModel):
     """
 
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
-    user_id = models.PositiveIntegerField(null=False, blank=False)
```

Comment:
If we must depend on `django.contrib.auth.User`, and we do not want to have references everywhere, then we could introduce a "user data service" (eg, `services/user_data.py`) and hide references to auth.User behind this abstraction.  This would be helpful for when `enterprise` is moved to its own IDA and the user data becomes truly remote.  This pattern could be expanded to cover many remote services for future workflow orchestration -- registration, enrollment, grading, data sharing, etc...